### ISSUE TITLE
'SecurityProtocolType.Ssl3' is obsolete

### ISSUE THREAD

**Issue by tmenezes** [flagship-io/flagship-dotnet-sdk#6]
The version 3.0 of the SDK is using SSL 3 which is obsolete and not supported anymore.
I'm not able to use the SDK due to the exception below.
I'm following code sample from [documentation](https://docs.developers.flagship.io/docs/net-v30x#advanced-configuration).

Exception:
```
Unhandled exception. System.NotSupportedException: The requested security protocol is not supported.
```

Compiler warning.
```
Severity	Code	Description	Project	File	Line	Suppression State
Warning	CS0618	'SecurityProtocolType.Ssl3' is obsolete: 'This value has been deprecated.  It is no longer supported. https://go.microsoft.com/fwlink/?linkid=14202'
```

**Comment by David-Flagship** [comment]
Hi @tmenezes, thanks for the warn 👏, we'll tell our devs about this and make our possible to resolve this one

**Comment by Heihokon** [comment]
Fixed in version [3.0.1](https://www.nuget.org/packages/Flagship.SDK/3.0.1)

Thanks for your contribution
